web/src/components/Organization/OrganizationPickListsCell/OrganizationPickListsCell.tsx

```diff
@@ -37,20 +38,7 @@ export const Success = ({
 }: CellSuccessProps<FindOrganizationQuery, FindOrganizationQueryVariables>) => {
   return (
     <div>
-      <Label
-        name="reportingPeriodId"
-        className="rw-label"
-        errorClassName="rw-label rw-label-error"
-      >
-        Reporting Period
-      </Label>
-      <SelectField name="reportingPeriodId">
-        {organization.reportingPeriods.map((reportingPeriod) => (
-          <option key={reportingPeriod.id} value={reportingPeriod.id}>
-            {reportingPeriod.name}
-          </option>
-        ))}
-      </SelectField>
+      <HiddenField name="reportingPeriodId" value={organization.preferences.current_reporting_period_id} />
```

Comment:
Just confirming, would check this in browser if you haven't already. Trying to read the JSON like this caused an error for me and I had to cast `preference` to a `JSONObject` (I think I had to cast the id too actually, like `((organization.preferences as Prisma.JsonObject).current_reporting_period_id as number)` or something ridiculous. If this works that's way better though!

Reply:
thanks for calling this out, I just tested locally and confirming that this works when I attempt to upload a file:
<img width="842" alt="image" src="https://github.com/usdigitalresponse/cpf-reporter/assets/6497366/ae9871e0-0168-4ba3-84de-ea3a1f1330ad">